lib/widgets-page.php

```diff
@@ -69,12 +69,16 @@ function gutenberg_widgets_init( $hook ) {
 		);
 	}
 
+	$experiments_exist        = get_option( 'gutenberg-experiments' );
```

Comment:
Hi @tellthemachines instead of replicating logic from function gutenberg_experiments_editor_settings here could we just call it on return?
```
return gutenberg_experiments_editor_settings( $settings );
```
This would also take care of enabling navigation on the widgets screen, would remove duplicate code.

I followed this approach in https://github.com/WordPress/gutenberg/pull/17005.

Reply:
Done!